src/Resources/assets/controllers/accordion_controller.js

```diff
@@ -1,70 +1,67 @@
 import { Controller } from '@hotwired/stimulus';
-import * as StickyThead from 'stickythead'
 
 export default class extends Controller {
-    static targets = ['content', 'arrow']
+    static targets = ['header', 'content', 'arrow']
+    static classes = ['arrowRotate', 'contentHidden']
 
-    toggle(event) {
-        const current = event.currentTarget;
-        const arrow = current.querySelector('[data-araise--table-bundle--accordion-target=arrow]');
-        const isOpen = current.dataset.ariaExpanded == 'true';
-        const nextSiblings = this.nextUntil(current, '[data-action="click->araise--table-bundle--accordion#toggle"]');
-
-        if(isOpen) {
-            arrow.classList.remove('rotate-90');
-            current.dataset.ariaExpanded = 'false';
-
-            nextSiblings.forEach((sibling) => {
-                sibling.classList.add('hidden');
-            });
-        } else {
-            arrow.classList.add('rotate-90');
-            current.dataset.ariaExpanded = 'true';
-
-            nextSiblings.forEach((sibling) => {
-                sibling.classList.remove('hidden');
-            });
-        }
-    }
-
-    /*!
-     * Get all following siblings of each element up to but not including the element matched by the selector
-     * (c) 2017 Chris Ferdinandi, MIT License, https://gomakethings.com
-     * @param  {Node}   elem     The element
-     * @param  {String} selector The selector to stop at
-     * @param  {String} filter   The selector to match siblings against [optional]
-     * @return {Array}           The siblings
+    /**
+     * @param {Event} event
      */
-    nextUntil(elem, selector, filter) {
-
-        // Setup siblings array
-        var siblings = [];
+    toggle(event)
+    {
+        /** @type {HTMLElement[]} */
+        const contentTargets = this.contentTargets;
 
-        // Get the next sibling element
-        elem = elem.nextElementSibling;
+        /** @type {HTMLElement[]} */
+        const headerTargets = this.headerTargets;
 
-        // As long as a sibling exists
-        while (elem) {
+        /** @type {HTMLElement[]} */
+        const arrowTargets = this.arrowTargets;
 
-            // If we've reached our match, bail
-            if (elem.matches(selector)) break;
+        /** @type {string} */
+        const arrowRotateClassString = this.arrowRotateClasses;
 
-            // If filtering by a selector, check if the sibling matches
-            if (filter && !elem.matches(filter)) {
-                elem = elem.nextElementSibling;
-                continue;
-            }
-
-            // Otherwise, push it to the siblings array
-            siblings.push(elem);
-
-            // Get the next sibling element
-            elem = elem.nextElementSibling;
+        /** @type {string} */
```

Comment:
`this.[logicalName]Classes` = Array
`this.[logicalName]Class` = String

```suggestion
        /** @type {string[]} */
```